src/selectors/visibleColumnBreakpoints.js

```diff
@@ -101,10 +102,20 @@ export function getColumnBreakpoints(pausePoints, breakpoints, viewport) {
     ({ location: { line } }) => lineCount[line] > 1
   );
 
-  return columnBreakpoints.map(({ location }) => ({
-    location,
-    enabled: isEnabled(location, breakpointMap)
-  }));
+  return columnBreakpoints.map(({ location }) => {
+    const foundBreakpoint = breakpoints.find(breakpoint => {
```

Comment:
I think it is fine as a first pass, but as column bps move forward, I'm going to be very curious to see how often this fails to match. I could easily see our breakpoints sliding a causing thins to fail to find the condition.